include/kframework/lemmas/lemmas.k

```diff
@@ -91,12 +91,14 @@ module LEMMAS [symbolic]
       [simplification]
 
  // Hardcoded #addrFromPrivateKey simplifications, see: https://github.com/runtimeverification/haskell-backend/issues/3573
-    rule #addrFromPrivateKey("0x0000000000000000000000000000000000000000000000000000000000000001") => 721457446580647751014191829380889690493307935711 [priority(40)]
+    rule #addrFromPrivateKey("0x0000000000000000000000000000000000000000000000000000000000000001") => 721457446580647751014191829380889690493307935711 [priority(40), concrete]
```

Comment:
The LEMMAS module is symbolic, so if this rule is marked concrete, it'll be ignored by both backends and is effectively dead code. Or am I misunderstanding something?

Reply:
No, `concrete` doesn't mean that as an attribute on rules. It instead means "only apply this rule if all the arguments are `concrete`". Doesn't affect whichbackends see the rule.

Reply:
Ah okay - we should update the K User Manual to better document this. The current documentation only mentions that behavior specifically for simplification rules: https://kframework.org/docs/user_manual/#concrete-and-symbolic-attributes-(haskell-backend)